components/schemes/Color/README.md

```diff
@@ -0,0 +1,34 @@
+<!--docs:
```

Comment:
Needs copyright stanza?

Reply:
Not in README's.

Reply:
Do we know the copyright status of the text written within a README if it's not covered by the same license as the prose in our source (comments)?

Reply:
This is out of scope for this PR, but worth discussing.